apprentice_teacher.rb

```diff
@@ -1,51 +1,17 @@
-class ApprenticeTeacher
+require_relative 'SeniorTeacher'
+
+class ApprenticeTeacher < SeniorTeacher
```

Comment:
An apprentice teacher is a senior teacher? That doesn't sound quite right to me....